builder/FindGlobals.cmake

```diff
@@ -79,20 +78,20 @@ if( Linux OR Darwin )
     set( git_commit "" )
     git_describe( git_commit )
 
-    add_definitions( -DMSDK_BUILD=\"$ENV{BUILD_NUMBER}\")
+    set( common_flags "${common_flags} -DMSDK_BUILD=\\\"$ENV{BUILD_NUMBER}\\\"" )
   endif()
 
   if (CMAKE_C_COMPILER MATCHES icc)
     set(no_warnings "-Wno-deprecated -Wno-unknown-pragmas -Wno-unused")
   else()
     set(no_warnings "-Wno-deprecated-declarations -Wno-unknown-pragmas -Wno-unused")
   endif()
-
+ 
   set(c_warnings "-Wall -Wformat -Wformat-security")
   set(cxx_warnings "${c_warnings} -Wnon-virtual-dtor")
 
-  set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -pipe -fPIC ${c_warnings} ${no_warnings}")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pipe -fPIC ${cxx_warnings} ${no_warnings}")
+  set(CMAKE_C_FLAGS   "${common_flags} ${CMAKE_C_FLAGS} -pipe -fPIC ${c_warnings} ${no_warnings}")
```

Comment:
add in the very end, please. let's use the scheme:
`set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}   <new stuff>")`

Reply:
done